all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -57,7 +57,7 @@ objects:
       spec:
           containers:
             - name: ${JAEGER_NAME}
-              image: jaegertracing/all-in-one:f209812d
+              image: jaegertracing/all-in-one:9f3984cd
```

Comment:
I don't really think that using snapshot versions is something safe for public templates. E.g. latest+ifNotPresent would not do any change on Rafael's notebook and maybe many people on conferences trying MSA.

There are currently tags for each commit in master, we talked to @yurishkuro about this (via email, I believe you were in cc) and it should be safe to use them as an intermediate solution until a proper version if out.